nicegui/json/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""
+Custom json module. Provides dumps and loads implementations
+wrapping orjson.
+
+Custom module required in order to override json-module used
+in socketio.AsyncServer.
+"""
+
+from typing import Any
+
+import orjson
+from fastapi import Response
+
+ORJSON_OPTS = orjson.OPT_SERIALIZE_NUMPY | orjson.OPT_NON_STR_KEYS
+
+
+def dumps(obj, sort_keys=False, separators=None):
```

Comment:
Can we add type annotation for `dumps` and `loads`?

Reply:
Done.